sentry/src/main/java/io/sentry/protocol/SdkVersion.java

```diff
@@ -23,7 +25,7 @@
    * <p>Official Sentry SDKs use the entity `sentry`, as in `sentry.python` or
    * `sentry.javascript.react-native`. Please use a different entity for your own SDKs.
    */
-  private String name;
```

Comment:
If we make these fields not nullable, they have to be either required in the constructor on they have to have default non-null values. Otherwise there is a mismatch when `new SdkVersion()` is executed.

Reply:
good catch, we can't make it final though because of JSON deserialization